src/sinks/influxdb_metrics.rs

```diff
@@ -0,0 +1,1056 @@
+use crate::{
+    event::metric::{Metric, MetricValue},
+    sinks::util::{
+        http::{Error as HttpError, HttpRetryLogic, HttpService, Response as HttpResponse},
+        BatchEventsConfig, MetricBuffer, SinkExt, TowerRequestConfig,
+    },
+    topology::config::{DataType, SinkConfig, SinkContext, SinkDescription},
+};
+use chrono::{DateTime, Utc};
+use futures::{Future, Poll};
+use http::{Method, StatusCode, Uri};
+use hyper;
+use hyper_tls::HttpsConnector;
+use lazy_static::lazy_static;
+use serde::{Deserialize, Serialize};
+use snafu::ResultExt;
+use std::cmp::Ordering;
+use std::collections::{BTreeMap, HashMap};
+use tower::Service;
+
+pub enum Field {
+    /// string
+    String(String),
+    /// float
+    Float(f64),
+    /// unsigned integer
+    UnsignedInt(u32),
```

Comment:
It seems like these should be 64 bit? https://v2.docs.influxdata.com/v2.0/reference/syntax/line-protocol/#uinteger

Reply:
It's `u32` because is used in `MetricValue` - https://github.com/timberio/vector/blob/61f9f3356bfdc266d6841d42344dd5dee194b766/src/event/metric.rs#L24